### PR TITLE
Package getopt.20230213

### PR DESCRIPTION
**Pull request by scemama**
### `getopt.20230213`
Parsing of command line arguments (similar to GNU GetOpt) for OCaml
General command line syntax of GNU getopt and getopt_long, but is close to the spirit of the Arg module.

---
* Homepage: https://github.com/scemama/ocaml-getopt
* Source repo: git+https://github.com/scemama/ocaml-getopt.git
* Bug tracker: https://github.com/scemama/ocaml-getopt/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

**Comment by scemama** [comment]
I have sent an invitation to @alainfrisch to join this repository.

**Comment by mseri** [comment]
@alainfrisch links to https://github.com/gildor478/ocaml-getopt as the official repository. Ping also @glidor478.

Are you all ok with this fork or you would prefer a different arrangement?

**Comment by scemama** [comment]
> @alainfrisch links to https://github.com/gildor478/ocaml-getopt as the official repository. Ping also @glidor478.
OK! Thanks for the info.

> Are you all ok with this fork or you would prefer a different arrangement?
I will try to talk with @glidor478 to see what is the best solution, but I would prefer him to be the maintainer of the package.

**Comment by scemama** [comment]
I exchanged emails with @gildor478. He told me he is not active anymore in the OCaml community and prefers that I host the repo.
So if everything is OK on your side, we can merge.

**Comment by mseri** [comment]
Thanks for looking into it. It would be useful to have the repository by @gildor478 archived with a note in the README pointing at your repo to clear doubts in the future

File: packages/getopt/getopt.20230213/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis:
+  "Parsing of command line arguments (similar to GNU GetOpt) for OCaml"
+description:
+  "General command line syntax of GNU getopt and getopt_long, but is close to the spirit of the Arg module."
+maintainer: ["Alain Frisch"]
```
Comment (mseri) [inline packages/getopt/getopt.20230213/opam:7]:
```suggestion
maintainer: ["Alain Frisch"]
```
You should probably add your name here if you have become the maintainer

Comment (scemama) [reply packages/getopt/getopt.20230213/opam:7]: Done!

**Comment by scemama** [comment]
> Thanks for looking into it. It would be useful to have the repository by @gildor478 archived with a note in the README pointing at your repo to clear doubts in the future

I asked him to do so by email, and added a note in my README:
> Note: This repository was introduced for compatibility with OCaml 5.0 in Feb 2023.
> Before, getopt was packaged with Oasis and was hosted here: https://github.com/gildor478/ocaml-getopt

**Comment by scemama** [comment]
To keep the history, I will fork @gildor478's repo and apply my changes to it. Please wait a bit before merging, I do it now.

**Comment by scemama** [comment]
Done! You can merge :-)

**Comment by mseri** [comment]
Thanks

**Comment by mseri** [comment]
Thanks a lot!

**Pull request merged**